src/Quaternions.jl

```diff
@@ -4,8 +4,8 @@ module Quaternions
 
   import Base: +, -, *, /, ^
   import Base: abs, abs2, angle, conj, cos, exp, inv, isfinite, log, real, sin, sqrt
-  import Base: convert, promote_rule, float
-  import LinearAlgebra: norm, normalize
+  import Base: convert, promote_rule, float, imag
```

Comment:
No need to import `imag`, it is exported from `Base`.

Reply:
changed to .im instead of imag - seems like there's an issue with importing imag
https://github.com/JuliaGeometry/Quaternions.jl/issues/29